feathr_project/feathr/udf/_preprocessing_pyudf_manager.py

```diff
@@ -53,6 +53,8 @@ def build_anchor_preprocessing_metadata(anchor_list: List[FeatureAnchor], local_
                 continue
             preprocessing_func = anchor.source.preprocessing
             if preprocessing_func:
+                print("Preprocessing function defined")
```

Comment:
Should this debug code get removed before merge?

Reply:
Good catch, fixed. It was left over from main branch in my fork.

Reply:
Yes, removed.